nn/source/hicFileReader.py

```diff
@@ -77,13 +78,15 @@ def read_data(self, debug_mode = False, noDump = False, fill_empty_contacts = Fa
 
         def get_data_straw006():
             try:
+
                 return straw.straw(self.normalization, self.fname,
                                      chr, chr, "BP", self.binsize)
             except TypeError:
                 if "chr" in chr:
                     new_chr = chr.replace("chr", "", 1)
                     logging.getLogger(__name__).warning("Failed to find chr " + chr + "; trying to find " + new_chr)
                     if new_chr in chr:
+                        print(new_chr)
```

Comment:
Do we really need print here? Why? If yes, please rewrite using logger architecture